modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/VerticalNav.js

```diff
@@ -36,13 +37,18 @@ export default function VerticalNav({
 			active={active}
 			className={cssClass}
 			decorated={decorated}
+			defaultExpandedKeys={new Set(defaultExpandedKeys)}
```

Comment:
Do we need to set the default value? I believe [it is set in Clay](https://github.com/liferay/clay/blob/master/packages/clay-core/src/vertical-nav/VerticalNav.tsx#L138).

Reply:
I tried without it and it was working, is it a problem?

Reply:
Oh I see, this is a List that serializes into an array, breaking in Clay. Nvm, this is fine :+1: